src/modules/classTaskList.js

```diff
@@ -0,0 +1,77 @@
+/*eslint-disable*/
+import Task from './classTask';
+import isStorageAvailable from './localStorage';
+
+const storageName = 'taskList';
+
+export default class TaskList {
+  constructor() {
+    this.data = [];
+    this.id = 1;
+    this.initUpdate();
+  }
+
+  initUpdate() {
+    if (isStorageAvailable('localStorage')) {
+      const data = JSON.parse(localStorage.getItem(storageName));
+
+      if (data && data.length !== 0) {
+        this.data = JSON.parse(localStorage.getItem(storageName));
+
+        const lastItem = this.data.reduce((prev, current) => {
+          const val = prev.id > current.id ? prev : current;
+          return val;
+        });
+        this.id = lastItem.id + 1;
+      }
+    }
+  }
+
+  addTask(task) {
+    if (isStorageAvailable) {
+      const taskObj = new Task(task, this.id);
+
+      this.data.push(taskObj);
+      localStorage.setItem(storageName, JSON.stringify(this.data));
+
+      this.id += 1;
+    }
+  }
+
+  removeTask(taskID) {
+    this.data = this.data.filter((i) => i.id !== taskID);
+    this.resetIds();
+    localStorage.setItem(storageName, JSON.stringify(this.data));
+  }
+
+  updateStatus(taskID, status) {
+    this.data = this.data.map((obj) => {
+      if (obj.id === taskID) {
+        return { ...obj, completed: status };
+      }
+
+      return obj;
+    });
+    localStorage.setItem(storageName, JSON.stringify(this.data));
+  }
```

Comment:
- [ ] When click on the checkbox completed should be true. Kindly fix it.

![Screenshot from 2022-07-09 00-23-54](https://user-images.githubusercontent.com/38635158/178072926-b906ac59-8fa6-4083-9df0-91dfc1e6769c.png)